enumerable.rb

```diff
@@ -0,0 +1,124 @@
+module Enumerable
+  def my_each
+    return to_enum unless block_given?
+
+    my_array = is_a?(Range) ? to_a : self
+
+    my_array.size.times { |n| yield (my_array[n]) }
+    self
+  end
+
+  def my_each_with_index
+    return to_enum unless block_given?
+
+    my_array = is_a?(Range) ? to_a : self
+
+    my_array.size.times { |n| yield my_array[n], n }
+    self
+  end
```

Comment:
### #my_each_
* [ ] with_index when a block is given and self is a hash behaves like #each_with_index calls the given block once for each element in self

Reply:
ALREADY WORKS !!!!!!! THIS PASSED THE EXAMPLE TEST
![error2](https://user-images.githubusercontent.com/51087315/83947096-937a3880-a81d-11ea-9f38-d4d24979c2f8.jpg)
 FROM THE RUBY DOCUMENTATION, WHAT OTHER TEST IS IT SUPPOSED TO PASS???